### DISCUSSION TITLE
/v2/md-enhance/zh/guide/demo/react.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#163] [Announcements]
# /v2/md-enhance/zh/guide/demo/react.html

VuePress 的 Markdown 增强插件

https://vuepress-theme-hope.github.io/v2/md-enhance/zh/guide/demo/react.html

**Comment by coulsonzero** [comment]
这个功能很好，但有些渲染好像暂时加载不出来，如果可以完善下就更棒了， 总体功能很强大！
自定义css样式后还不错，如果有兴趣的可以查看https://docs.coulsonzero.top修改样式

**Reply by Mister-Hope** [reply]
React 和 Vue 需要等待 cdn 加载完成才能渲染，我看你的二次修改很不错。欢迎 PR 你的样式过来。

**Reply by Mister-Hope** [reply]
React demo improved in https://github.com/vuepress-theme-hope/vuepress-theme-hope/commit/7008759ff4adf88e174d38e1d74933dfd3338505
